content/docs/tasks/telemetry/servicegraph/index.md

```diff
@@ -1,10 +1,13 @@
 ---
-title: Generating a Service Graph
+title: Generating a Service Graph*
 description: This task shows you how to generate a graph of services within an Istio mesh.
 weight: 50
 keywords: [telemetry,visualization]
 ---
 
+> \* Note: The Service Graph has been deprecated.
+> It has been replaced with [Kiali](../kiali).
```

Comment:
this note in general should go in the release notes:

https://preliminary.istio.io/about/notes/